### PR TITLE
_WD_ExecuteScript: don't reformat JS code

### LINKED ISSUE #487: issue in _WD_ExecuteScript / __WD_EscapeString when comments // was used in JavaScript string
### Describe the bug
`$sData = StringRegExpReplace($sData, '[\v\t]', '')`
from here:
https://github.com/Danp2/au3WebDriver/blob/42dda7d9a9e78b3c7cc2d419b0e84eaf1335f6a8/wd_core.au3#L1747-L1749

strips `[\v]` chars and when `// comment` is used entire `$sJavaScript` becames commented out.

### How to reproduce

```autoit
_Example()
Func _Example()
	Local $sJavaScript = _
			"// comment" & @CRLF & _
			"return document.readyState;" & @CRLF & _
			""
	_WD_DebugSwitch($_WD_DEBUG_Full)
	_WD_ExecuteScript($sSession, $sJavaScript)
	_WD_DebugSwitch()
EndFunc   ;==>_Example
```

### Expected behavior
get result from: `return document.readyState;`

### Screenshots
none

### Additional context

https://www.autoitscript.com/forum/topic/208640-webdriver-udf-help-support-iv/?do=findComment&comment=1521863

### System under test

not related

### PR DESCRIPTION
**Pull request by Danp2**
### Proposed changes

Fixes #487 

### Checklist

Put an `x` in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We are here to help!<br>
This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read and noticed the [CODE OF CONDUCT](https://github.com/Danp2/au3WebDriver/blob/master/docs/CODE_OF_CONDUCT.md) document
- [x] I have read and noticed the [CONTRIBUTING](https://github.com/Danp2/au3WebDriver/blob/master/docs/CONTRIBUTING.md) document
- [ ] I have added necessary documentation or screenshots (if appropriate)

### Types of changes

Please check `x` the type of change your PR introduces:

- [x] Bugfix (change which fixes an issue)
- [ ] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (functional, structural)
- [ ] Documentation content changes
- [ ] Other (please describe)

### What is the current behavior?

Stripping of tabs and spaces from JS can result in code that no longer functions properly.

### What is the new behavior?

Code is processed as-is. Any reformatting that is required should be performed prior to calling _WD_ExecuteScript

### CODE REVIEW

**Comment by mlipok** [comment]
still need to check, but currently focusing on frames issue.

**Comment by mlipok** [comment]
> still need to check, but currently focusing on frames issue.

here is my latest opinion

https://github.com/Danp2/au3WebDriver/issues/487#issuecomment-1659380663

**Comment by mlipok** [comment]
Please remove all `$JSON_MLREFORMAT` usage

**Comment by mlipok** [comment]
Also in `wd_helper.au3` the use of `StringReplace(...... , @TAB, '')` should be considered a candidate for removal.

**Comment by mlipok** [comment]
> Also in `wd_helper.au3` the use of `StringReplace(...... , @TAB, '')` should be considered a candidate for removal.

because:
```autoit
$sData = Json_StringEncode($sData, $iOption) ; Escape JSON Strings
```
from here:
https://github.com/Danp2/au3WebDriver/blob/c39d1f56fee86a5ebe5c9a9f9082ccdea120a5f2/wd_core.au3#L1744C1-L1748C32

It's enough

**Comment by mlipok** [comment]
The other question is why we always return `$_WD_ERROR_Success` here:
https://github.com/Danp2/au3WebDriver/blob/c39d1f56fee86a5ebe5c9a9f9082ccdea120a5f2/wd_core.au3#L1747

**Comment by mlipok** [comment]
> Also in `wd_helper.au3` the use of `StringReplace(...... , @TAB, '')` should be considered a candidate for removal.

but maybe not? I think they are ther to CleanUp the mess in sphagetthi called logs.

File: wd_core.au3 [hunk 1]
```diff
@@ -1738,18 +1736,12 @@ EndFunc   ;==>__WD_CloseDriver
 ;                  $iOption - [optional] Any combination of $JSON_* constants. Default is 0.
 ; Author ........: Danp2
 ; Modified ......:
-; Remarks .......: $JSON_MLREFORMAT will strip tabs and CR/LFs from a multiline string.
-;                  See $JSON_* constants in json.au3 for other $iOption possibilities.
+; Remarks .......:
```
Comment (mlipok) [inline wd_core.au3:1739]: this remark
`See $JSON_* constants in json.au3 for other $iOption possibilities.`

should stay here.

**Comment by Danp2** [comment]
> Also in wd_helper.au3 the use of StringReplace(...... , @TAB, '') should be considered a candidate for removal.

To me, these are two separate issues:

1) JS code submitted by the user
2) JS code supplied internally by the UDF

This PR is intended to only deal with the 1st instance. 

> but maybe not? I think they are there to CleanUp the mess in sphagetthi called logs.

Right... IIRC, you wanted the inline code to be more legible so this is the way it was done.

**Comment by Danp2** [comment]
> The other question is why we always return $_WD_ERROR_Success here:

IDK. We are dealing with an internal function, so it may not matter. This appears to be a carry over from before we switched to using `Json_StringEncode`. I suppose we could check the error code from `Json_StringEncode`, but then return a different error code if the function returned an error.

FWIW, I also noticed that the function header is missing a `Return values` entry.

**Comment by mlipok** [comment]
> The other question is why we always return `$_WD_ERROR_Success` here:
> 
> https://github.com/Danp2/au3WebDriver/blob/c39d1f56fee86a5ebe5c9a9f9082ccdea120a5f2/wd_core.au3#L1747

This is the last question here

**Comment by mlipok** [comment]
> . I suppose we could check the error code from `Json_StringEncode`, but then return a different error code if the function returned an error.

Please propose solution.I will check when I woke up.

**Comment by mlipok** [comment]
Looks fine for me

**Pull request merged**